MapboxCoreNavigation/NavigationServiceDelegate.swift

```diff
@@ -30,7 +30,7 @@ public protocol NavigationServiceDelegate: class, UnimplementedLogging {
     /**
      Called immediately before the navigation service calculates a new route.
      
-     This method is called after `navigationService(_:shouldRerouteFrom:)` is called, simultaneously with the `NavigationServiceWillReroute` notification being posted, and before `navigationService(_:didRerouteAlong:)` is called.
+     This method is called after `navigationService(_:shouldRerouteFrom:)` is called, simultaneously with the `Notification.Name.routeControllerWillReroute` notification being posted, and before `navigationService(_:didRerouteAlong:)` is called.
```

Comment:
A bunch of these references were always inaccurate. I don’t think we’ve ever prefixed the notification names with `NavigationService`.